_data/simple-icons.json

```diff
@@ -13829,6 +13829,17 @@
             "source": "https://www.viber.com/brand-center",
             "guidelines": "https://www.viber.com/brand-center"
         },
+        {
+            "title": "VictoriaMetrics",
+            "aliases": {
+                "aka": [
+                    "VM"
+                ]
+            },
+            "hex": "110F0F",
+            "source": "https://github.com/VictoriaMetrics/VictoriaMetrics/blob/26cee364991ebec76cf3ebcf0b28beb66e0ff53e/VM_logo.zip",
```

Comment:
```suggestion
            "source": "https://github.com/VictoriaMetrics/VictoriaMetrics/blob/24d61bf19374b42ef9839c13c7d35ce8888170e0/docs/assets/images/vm_logo.svg",
```

Reply:
Ok, I'll update it :)

Purple(`#621773`) looks good